source/_data/glossary.yml

```
@@ -271,8 +271,25 @@
   link: /docs/blueprint/selectors/
 
 - term: Service
-  definition: >-
-    Services are called to perform actions.
+  definition: |-
+    The term service has 2 meanings in Home Assistant:
+
+    **The information service**
+    E.g. the municipal waste management service that provides entities for organic, paper, and packaging waste. In terms of functionality, the information service is like a {% term device %}. It is called *service* to avoid confusion, as it does not come with a piece of hardware.
```

```suggestion
    E.g. the municipal waste management service that provides entities for organic, paper, and packaging waste. In terms of functionality, the information service is like a device. It is called *service* to avoid confusion, as it does not come with a piece of hardware.
```

Plugins don't work here... not sure why. Made a note, looking at it later